xml/art_solution_monitoring.xml

```diff
@@ -0,0 +1,251 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="file:/usr/share/xml/geekodoc/rng/geekodoc5-flat.rnc"
+   type="application/relax-ng-compact-syntax"?>
+
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<!--
+Further information:
+
+* SAP Monitoring will become easier and powerful using SLES for SAP
+  https://suse.com/c/coming-soon-sap-landscape-monitoring-on-sles-for-sap/
+-->
+
+<article version="5.1" xml:lang="en" xml:id="art-sol-monitoring"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+   <title>&sap; Monitoring</title>
+   <info>
+     <abstract>
+       <para>
+         This article shows monitoring solutions for &sap; administrators
+         to efficiently monitor their &sap; systems.
+       </para>
+       <para>
+         The solution described herein is not officially supported by &suse;.
+         Nevertheless, it is a solution based on best-efforts and published
+         <quote>as is</quote>.
+       </para>
+     </abstract>
+   </info>
+
+  <sect1 xml:id="sec-sol-monit-concept-overview">
+    <title>Conceptual Overview</title>
+    <para>
+      Starting from the idea to improve user experience, &suse; engineering worked
+      on how to monitor and troubleshoot clusters.
+    </para>
+    <para>
+      The exporters showed here can be combined and integrated with Prometheus
+      and Grafana to produce complex dashboards.
+    </para>
+    <para>
+      &suse; supports Prometheus and Grafana through &suma; 4.0. Some Grafana
+      dashboards for &hana;, &s4h;, &netweaver;, and the cluster monitoring
+      are provided by &suse; via Grafana community dashboards.
+    </para>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-monit-terms">
+    <title>Terminology</title>
+    <para></para>
+    <variablelist>
+      <varlistentry>
+        <term>Grafana</term>
+        <listitem>
+          <para>
+            An interactive, visualization and analytics web application.
+            With dashboards, it provides methods to visualize, explore,
+            alert on, and query your metrics.
```

Comment:
I find the phrasal verb in the middle of a list of words jarring.

How about:
"methods to visualize, explore, and query your metrics, and trigger alerts."
...?

Reply:
That's wonderful, thanks! :+1:  I'll use your sentence.